templates/sysbench_svc.py

```diff
@@ -32,6 +33,8 @@ def __init__(
         self.sysbench = f"/usr/bin/sysbench {tpcc_script} --threads={threads} --tables={tables} --scale={scale} --force_pk=1 --db-driver={db_driver} --report-interval=10 --time={duration} "
```

Comment:
I could be wrong, but IMHO `--force_pk=1` works for MySQL only.

Reply:
Thx, fixed on [this PR](https://github.com/canonical/sysbench-operator/pull/4)